src/ethereum/frontier/vm/memory.py

```diff
@@ -0,0 +1,92 @@
+"""
+Ethereum Virtual Machine (EVM) Memory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Implementation of the EVM memory expansion related operations.
+"""
+
+from typing import cast
+
+from ethereum.base_types import U256
+
+from . import Evm
+from .gas import GAS_MEMORY, subtract_gas
+
+
+def get_words(memory_size: int) -> int:
+    """
+    Obtain the number of words (32 bytes) which is present in the memory,
```

Comment:
> which is present in the memory

not sure this is true, since it's only operating on the input parameter

Reply:
This won't be used anymore as the memory operations will be removed from this PR in favour of #250